tests/pods/basic.bats

```diff
@@ -10,6 +10,13 @@ load ../helpers/k8s
 load ../helpers/run
 load ../helpers/sysbox-health
 
+function setup() {
```

Comment:
Why are you doing this? There's no option to install cri-o in amazon-linux?

Reply:
No, the CRI-O installation instructions don't support AL2. We would need to build from source if we want to run CRI-O in the test container on AL2 (in a future commit).